### DISCUSSION TITLE
Error setting up the container

### DISCUSSION THREAD

**Discussion by kibangadas** [tudelft/AE4353-Y24#13] [Q&A]
I was able to reach the last stage of the Usage process, however when I try to set up the container, it gets stuck on the last step:

[](url)
![imagem_2024-09-12_151336121](https://github.com/user-attachments/assets/88571eca-59cd-44dc-85e8-533a0239c1c1)

Afterwards, an error message pops up both in the terminal and as a new window:

![imagem_2024-09-12_151429816](https://github.com/user-attachments/assets/d057bb95-a2dd-44a1-90f7-8e94fcd8f4ba)

I tried rebuilding without cache but nothing changed

**Comment by yilun-wu** [comment]
The error is due to a connection interruption to `repo.anaconda.com` when conda is trying to download the packages. Please try it again with a stable Internet connection.

**Reply by kibangadas** [reply]
I'm on campus using TU Delft's internet ;(

**Comment by Huizerd** [comment]
could it be that your firewall is (for some reason) blocking this particular connection? I saw some people with Norton antivirus with problems with their internet connection in either WSL or the container
